blender_addons/io_md5_exporter/io_export_md5.py

```diff
@@ -960,6 +975,50 @@ def invoke(self, context, event):
         
         save_md5(modelFilePath, animationFilePath)
         return{'FINISHED'}
+
+
+def findAnimation(animName):
+  for anim in bpy.data.actions:
+    if animName in anim:
+      flag=1
+      return anim
+
+  return null
+
+
+import json,os
+from collections import OrderedDict
+class ExportPrefabForTerasology(bpy.types.Operator):
+  bl_idname="export.prefab"
+  bl_label="Export a prefab for Terasology"
+
+  def invoke(self, context, event):
+    current_file_dir=os.path.dirname(__file__)
+    file_in_path=os.path.join(current_file_dir,"template.prefab")
+    with open(file_in_path, 'r') as inFile:
+      data = json.load(inFile, object_pairs_hook=OrderedDict)
+
+    ModelName=getSuggestedModelName()

```

Comment:
At least in java it is convention to have methods name start lower case and classes upper case. Since it is an exporter for a java program it might be a good idea to follow that convention. Especially not using upper case for a variable